bundle/manifests/authorino-operator.clusterserviceversion.yaml

```diff
@@ -314,6 +314,12 @@ spec:
                       - --leader-elect
                     command:
                       - /manager
+                    env:
+                      - name: DEFAULT_AUTHORINO_IMAGE
+                        valueFrom:
+                          configMapKeyRef:
```

Comment:
what is the rationale to get the image url from the config map? why not a plain env var? The CSV will be the source of truth. 

new authorino image -> requires new operator bundle -> new CSV.

Reply:
This is so we can rely on kustomize tooling to set the variable value dynamically when we run `make manifests` (also triggered on `make bundle`.) There's a `configMapGenerator` that picks up the dynamic value.

Alternative approaches we employ for other customisations are:
1. using `envsubst`, or
2. directly setting the value in the `config/manager/manager.yaml` with `yq`.

Happy to change if you prefer any of these other options.

Reply:
The config map is yet another resource of the bundle. When there is a new image, even though the CSV does not change (not entirely true, check my next comment), a new bundle is needed anyway because the config map changes. 

Your call. Just keep in mind that the image of authorino URL is also referenced in the CSV in `relatedImages` section, so the CSV needs to change anyway.

Reply:
I don't like adding the ConfigMap either. I'll refactor this.